### ISSUE TITLE
WHy are the weights of GAN loss being 0?

### ISSUE THREAD

**Issue by kangzhiq** [amjltc295/Free-Form-Video-Inpainting#45]
Hello,

Thank you for this great work!

While going through your code, I noticed that the weight for the GAN loss is 0: [code](https://github.com/amjltc295/Free-Form-Video-Inpainting/blob/19bfc6fbe626385be549ea2fc6c1ba7852e48e56/src/config.json#L77). It basically means that the generator is trained with the reconstruction loss, perceptual loss, etc, without the supervision of the discriminator.

I am not sure if it is a mistake or your have trained your model in this way. I will  try to train the model again with the GAN loss integrated. 

Any explanation or remark from you would be highly appreciated. Thank you in advance!

**Comment by kangzhiq** [comment]
Closing the issue since I found this clarification in the paper:

> Empirically we found that the model converges slower when di-
rectly trained as a whole. Thus, during training, we first pre-train the generator without the
TSMGAN loss until convergence, and then fine-tune with the TSMGAN.
